nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -228,13 +228,17 @@ PREFER_IPV4 = False
 RACK_ELEVATION_DEFAULT_UNIT_HEIGHT = 22
 RACK_ELEVATION_DEFAULT_UNIT_WIDTH = 220
 
-# Remote authentication support
-REMOTE_AUTH_ENABLED = False
-REMOTE_AUTH_BACKEND = "nautobot.core.authentication.RemoteUserBackend"
+# Global 3rd-party authentication settings
+EXTERNAL_AUTH_DEFAULT_GROUPS = []
+EXTERNAL_AUTH_DEFAULT_PERMISSIONS = {}
+
+# Remote auth backend settings
+REMOTE_AUTH_AUTO_CREATE_USER = False
 REMOTE_AUTH_HEADER = "HTTP_REMOTE_USER"
-REMOTE_AUTH_AUTO_CREATE_USER = True
-REMOTE_AUTH_DEFAULT_GROUPS = []
-REMOTE_AUTH_DEFAULT_PERMISSIONS = {}
+
+# Configure SSO, for more information see docs/configuration/authentication/sso.md
+SOCIAL_AUTH_POSTGRES_JSONFIELD = False
+SOCIAL_AUTH_URL_NAMESPACE = "sso"
```

Comment:
Let's please get rid of this setting, and just go ahead and proceed with explicitly adding the URL routes for `social_django` using the defaults in `urls.py`.